mesa/Person.py

```diff
@@ -21,7 +21,7 @@ class Person(Agent):
         'professional',
         'school']      
     
-    def __init__(self, m: mesaPROTON_OC):
+    def __init__(self, m:mesaPROTON_OC):
```

Comment:
thanks. I still don't really understand how this way of declaring types is strict. I use it unsystematically - we have to decide. #172.

Reply:
No doubt they make it easier to read the code. I think it is useful to use them when the choice is forced, as in this case.